cmd/fyne/internal/commands/build.go

```diff
@@ -65,6 +67,11 @@ func Build() *cli.Command {
 				Usage:       "Specify a name for the output file, default is based on the current directory.",
 				Destination: &b.target,
 			},
+			&cli.GenericFlag{
```

Comment:
Am I right that you can have only 1 custom field? I'm not familiar with `GenericFlag` but it seems like setting a second will overwrite the first...

Reply:
Actually, if you look into flags.go, keyValueFlag is actually storing multiple key/value pairs in a map and the cli parser will call Set as often as needed. So it will store as many custom metadata as you want :-)